Makefile

```diff
@@ -41,14 +41,17 @@ APP_VER		:= $(shell git describe 2> /dev/null || echo "unknown")
 REGISTRY_APP_NAME		:= a8registry
 CONTROLLER_APP_NAME		:= a8controller
 SIDECAR_APP_NAME		:= a8sidecar
+K8SRULES_APP_NAME		:= a8k8srulescontroller
 
```

Comment:
The reason was that the controller would behave just like the registry and sidecar, With adapter backends for a8api, k8sapi, consul API, etc.